pallets/runtime/tests/src/confidential_test.rs

```diff
@@ -73,3 +84,59 @@ fn range_proof_we() {
         true
     );
 }
+
+#[test]
+fn account_create_tx() {
+    ExtBuilder::default()
+        .build()
+        .execute_with(account_create_tx_we);
+}
+
+fn account_create_tx_we() {
+    let alice = AccountKeyring::Alice.public();
+    let alice_id = register_keyring_account(AccountKeyring::Alice).unwrap();
+
+    let valid_asset_ids: Vec<AssetId> = vec![1, 2, 3]
```

Comment:
`[1, 2, 3]` is supposed to represent the plain ticker ids. Is this a fair representation/assumption?

Reply:
Yeah it is fine